packages/cli/lib/DevServerManager.js

```diff
@@ -113,6 +121,10 @@ class DevServerManager {
           await serverInterface.cleanup();
         }
       });
+
+      if (!portManagerHasActiveServers()) {
```

Comment:
portManagerHasActiveServers is async so you'll need await here. The stop function is also async and we might want to wait for that so we can handle errors if there are any